tests/functional/spec/census-household.spec.js

```diff
@@ -100,6 +101,7 @@ describe('Census Household', function () {
 
         // visitors
         completeVisitorSection()
+        VisitorsCompleted.submit()
```

Comment:
There can be multiple visitor sections, the visitors completed only happens once after all visitors have been completed.